.circleci/config.yml

```diff
@@ -85,7 +96,10 @@ jobs:
 workflows:
   test_and_deploy:
     jobs:
-      - test
+      - test:
+          matrix:
+            parameters:
+              python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
We have this in SDK-tests, should not need here

Reply:
I did not get it. Can you please elaborate?
Isn't the idea if this ticket to run the test pipeline on all supported python versions?